lib/rubocop/cop/gusto/discouraged_gem.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Gusto
+      # Flags installation of discouraged gems (e.g., timecop) in Gemfiles and gemspecs.
+      #
+      # Configuration:
+      #   Gems: ['timecop']
+      #   MessagePerGem:
+      #     timecop: "Use Rails' time helpers (e.g., freeze_time, travel_to) instead of Timecop."
+      #
+      # This cop is intended to be enabled in Rails projects via config/rails.yml.
+      class DiscouragedGem < Base
+        MSG = "Avoid using the '%{gem}' gem. %{advice}"
+
+        RESTRICT_ON_SEND = %i(gem add_dependency add_development_dependency).freeze
+
+        def on_send(node)
+          check_gem_usage(node)
+        end
+
+        private
+
+        def check_gem_usage(node)
+          gem_name = extract_gem_name(node)
+          return unless gem_name
+          return unless discouraged_gems.include?(gem_name)
```

Comment:
```suggestion
          return unless discouraged_gems.include?(first_literal_string(node))
```
Probably better to inline that private method as guard clauses without assignments, and pull it back up to here